### ISSUE TITLE
[Feature Request] PHP Unit @test annotation (or not)

### ISSUE THREAD

**Issue by BackEndTea** [FriendsOfPHP/PHP-CS-Fixer#3274]
PHP Unit accepts the following as tests.
```php
    /**
     * @test
     */
    public function seeIfItWorks() {...}
```
or 
```php
    public function testSeeIfItWorks() {...}
```
I'd like to see a rule that enforces one or the other, so either remove ```test``` from the function name and add ```@test``` annotation, or remove ```@test``` annotation, and add ```test``` in front of the function name. (And also make sure the function is properly camel cased again.)

**Comment by keradus** [comment]
Hi.
I think it is a good idea indeed.
Please, could you raise a PR ?

**Comment by BackEndTea** [comment]
I'm playing around with it now, and i'll try and make a PR out of this when its ready.

**Comment by keradus** [comment]
great to hear ;)
be sure to also give https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.8/doc/COOKBOOK-FIXERS.md  a try

**Comment by BackEndTea** [comment]
Since this was merged in #3275 i think this can be closed
